### ISSUE TITLE
ModuleNotFoundError: No module named 'shopping'

### ISSUE THREAD

**Issue by gjae** [googleapis/google-api-python-client#1423]
Hi there!. i've a problem with shopping module from google-api-python-client, i attempted create a new venv environment then reinstall the package, but not work

#### Environment details

  - OS type and version:  ```bash $ uname -a: Linux [user] 4.15.0-139-generic #143~16.04.1-Ubuntu SMP Wed Mar 17 08:10:33 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux``` 
  - Python version: `3.8`
  - pip version: `pip --version pip 20.2.4 from /var/www/html/...venv//lib/python3.8/site-packages/pip (python 3.8)`
  - `google-api-python-client` version:  
  `Name: google-api-python-client
Version: 2.9.0
Summary: Google API Client Library for Python
Home-page: https://github.com/googleapis/google-api-python-client/
Author: Google LLC
Author-email: googleapis-packages@google.com
License: Apache 2.0
Location: /var/www/html/../venv/lib/python3.8/site-packages
Requires: uritemplate, httplib2, google-auth, google-api-core, google-auth-httplib2, six
Required-by:`

#### Steps to reproduce

  1. source venv/bin/activate
  2. python3.8 insert-product.py -insert

#### Code example

```python
from shopping.content import common

product = self.collection.find({"_id": mongoDB_ID }) 
product = list(product)
if len(product) > 0:
    uniqueId = common.get_unique_id()
    print(uniqueId)
```

#### Stack trace
```
Traceback (most recent call last):
  File "ovnix.py", line 10, in <module>
    from modules.modules import moduleRegisters as modules
  File "/var/www/html/.../venv/modules/modules.py", line 7, in <module>
    from .google_merchant_center.index import MerchantCenter
  File "/var/www/html/.../venv/modules/google_merchant_center/index.py", line 3, in <module>
    from shopping.content import common
ModuleNotFoundError: No module named 'shopping'

```

#### extras
```bash
pip freeze | grep -i google

google-api-core==1.30.0
google-api-python-client==2.9.0
google-auth==1.31.0
google-auth-httplib2==0.1.0
googleapis-common-protos==1.53.0
```

Thanks.

**Comment by parthea** [comment]
Hi @gjae ,

This issue appears to be specific to a [sample](https://github.com/googleads/googleads-shopping-samples/blob/master/python/shopping/content/common.py#L150) in [googleads/googleads-shopping-samples](https://github.com/googleads/googleads-shopping-samples/blob/master/python/README.md) and unrelated to `google-api-python-client`. Please create a new issue [here](https://github.com/googleads/googleads-shopping-samples/issues/new).  If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) to ensure a timely response. I'm going to close this issue but please re-open it if you have additional questions.
